app/src/main/java/org/gdg/frisbee/android/api/FirebaseDynamicLinksHubFactory.java

```diff
@@ -0,0 +1,34 @@
+package org.gdg.frisbee.android.api;
+
+import com.google.gson.FieldNamingPolicy;
+
+import org.gdg.frisbee.android.utils.Utils;
+
+import okhttp3.OkHttpClient;
+import retrofit2.Retrofit;
+import retrofit2.converter.gson.GsonConverterFactory;
+
+/**
+ * Created by unstablebrainiac on 26/5/17.
+ */
+
+public final class FirebaseDynamicLinksHubFactory {
```

Comment:
`Hub` is the name for the api for GDG-X. Do we need `Hub` naming here? I would prefer `FirebaseDynamicLinksFactory`